frame/staking/src/lib.rs

```diff
@@ -726,7 +726,7 @@ impl<T: Trait> SessionInterface<<T as frame_system::Trait>::AccountId> for T whe
 	}
 }
 
-pub trait Trait: frame_system::Trait {
+pub trait Trait: frame_system::Trait + SendTransactionTypes<Call<Self>> {
```

Comment:
```suggestion
pub trait Trait: SendTransactionTypes<Call<Self>> {
```

Reply:
`SendTransactionTypes` is no longer has `frame_system::Trait` as a super trait to enable submitting unsigned transaction without having to define `AppCrypto`. This is why i have this change here.

Reply:
Fair enough! I saw that `frame_system::Trait` is removed in some other place, but it was only because there is also a historical session bound.